src/common/components/Toast/utils/setToast.ts

```diff
@@ -0,0 +1,17 @@
+import { Bounce, toast } from "react-toastify";
+
+type TostType = "info" | "success" | "warn" | "error";
```

Comment:
타입 재활용을 해주시면 좋을 것 같아요~
```suggestion
type ToastType = keyof typeof ToastTextType;
```